src/styles/stars.css

```
@@ -0,0 +1,25 @@
+.stars__star {
+  background-image: url(../images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 24px;
+}
+
```

move block styles to top of the file
```suggestion
.stars {
  display: flex;
  margin-bottom: 24px;
}

.stars__star {
  background-image: url(../images/star.svg);
  width: 16px;
  height: 16px;
  background-repeat: no-repeat;
  background-position: center;
  margin-right: 4px;
}

.stars__star:last-child {
  margin-right: 0;
}

```